config/initializers/paths.rb

```diff
@@ -3,26 +3,14 @@ module PathsInitializer
 
   class << self
     def perform_initialization(config)
-      if Rails.env == "test"
-        config.data_root = Rails.root.join('tmp','data')
-      else
-        config.data_root = Rails.root.join('data').sub(%r(/releases/\d+/), "/current/")
-      end
+      settings = Rails.env == "test" ? nil : Settings.git
+      config.data_root = (settings && settings.data_dir) \
```

Comment:
Style/MultilineTernaryOperator: Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.